.vortex/installer/tests/Functional/FunctionalTestCase.php

```
@@ -116,12 +108,16 @@ protected function replaceVersions(string $dir): void {
       ->addVersionReplacements()
       // PHPStan phpVersion is an integer (e.g., 80330), not semver.
       ->addReplacement(Replacement::create('phpstan_version', '/(phpVersion:\s)\d{5,6}/', '${1}' . Replacement::VERSION))
+      // The Vortex badge carries the checked-out ref, so every fixture churns
+      // on a tagged checkout unless it is masked like any other version stamp.
+      ->addReplacement(Replacement::create('vortex_badge', '#(badge/Vortex-)[^-]+(-65ACBC\.svg)#', '${1}' . Replacement::VERSION . '${2}'))
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\AlexSkrypnyk\File\Replacer\Replacement' in method 'replaceVersions'. (undefined)

(StaticAccess)